src/charsmap_normalization.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "normalizer.h"  // from sentencepiece
+#include <openvino/op/op.hpp>
+
+
+using namespace ov;
+
+class CharsMapNormalization : public ov::op::Op {
```

Comment:
Could you please add docstring with brief description what this operation does?

Reply:
Added